samples/dashboards/data/mqtt-connector/demo.js

```diff
@@ -0,0 +1,645 @@
+/* *
+ *
+ *  Sample application using a MQTT connector (custom connector)
+ *
+ * */
+
+// Global board instance for use in event handlers
+let board = null;
+
+
+// Highcharts common options
+Highcharts.setOptions({
+    chart: {
+        type: 'spline',
+        animation: true
+    },
+    legend: {
+        enabled: false
+    },
+    xAxis: {
+        type: 'datetime',
+        labels: {
+            format: '{value:%H:%M}'
+        }
+    },
+    yAxis: {
+        title: {
+            text: 'Power (MW)'
+        }
+    },
+    tooltip: {
+        useHTML: true,
+        formatter: function () {
+            const date = Highcharts.dateFormat('%A, %b %e, %H:%M', this.x);
+            return `<b>${date} (UTC)</b><hr>Generated power: ${this.y} MW<br/>`;
+        }
+    }
+});
+
+const dataGridOptions = {
+    cellHeight: 38,
+    editable: false,
+    columns: {
+        time: {
+            headerFormat: 'Time (UTC)',
+            cellFormatter: function () {
+                const date = new Date(this.value);
+                return Highcharts.dateFormat('%H:%M', date);
+            }
+        },
+        power: {
+            headerFormat: 'Power (MW)'
+        }
+    }
+};
+
+// MQTT connection configuration
+const mqttConfig = {
+    host: 'mqtt.sognekraft.no',
+    port: 8083,
+    user: 'highsoft',
+    password: 'Qs0URPjxnWlcuYBmFWNK',
```

Comment:
Is this OK frontend?

Reply:
With a custom MQTT broker (or a public broker like Mosquitto) the login info will become redundant.